shared/constants/types/teams.tsx

```diff
@@ -116,8 +115,10 @@ export type InviteLink = {
 export type TeamDetails = {
   members: Map<string, MemberInfo>
   settings: TeamSettings2
-  invites: Set<InviteInfo>
-  inviteLinks: Set<InviteLink>
+  // Legacy invites that are guaranteed to be active
+  validInvites: Set<ValidInviteInfo>
```

Comment:
Prefer not to change this too. Is there a reason not to change this to array also?